assets/panel/debugger.properties

```diff
@@ -704,10 +704,10 @@ functionSearchSeparatorLabel=←
 
 # LOCALIZATION NOTE(gotoLineModal.placeholder): The placeholder
 # text displayed when the user searches for specific lines in a file
-# Do not localize "CmdOrCtrl+Shift+;", or change the format of the string. These are
+# Do not localize "CmdOrCtrl+;", or change the format of the string. These are
 # key identifiers, not messages displayed to the user.
 gotoLineModal.placeholder=Go to line…
-gotoLineModal.key=CmdOrCtrl+Shift+;
+gotoLineModal.key=CmdOrCtrl+;
```

Comment:
This should have been caught in review (apparently I missed there were new strings in here)

Reply:
good catch. I'll change this to `gotoLineModal.key2`